<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-7S7LCUO" failures="2" tests="6" name="Test" time="27.37" errors="0" timestamp="2024-04-02T16:51:52 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.878"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.927">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at parallel.Login.user_should_be_redirected_to_the_dashboard(Login.java:39)
at ✽.user should be redirected to the dashboard "2Test Automation Practice: Working with Secure Page"(file:///C:/Users/Hustler/eclipse-workspace/CucumberPOM/src/test/resources/parallel/Login.feature:21)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="19.167">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at parallel.Login.user_should_be_redirected_to_the_dashboard(Login.java:39)
at ✽.user should be redirected to the dashboard "1Test Automation Practice: Working with Login form"(file:///C:/Users/Hustler/eclipse-workspace/CucumberPOM/src/test/resources/parallel/Login.feature:27)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="19.361"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="20.016"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="20.062"/>
</testsuite> <!-- Test -->
